the-then-years-bug/the-ten-years-bug.md

```diff
@@ -1,13 +1,13 @@
 The Ten Years Bug
 =========================
 
-Developers often make mistakes. Bugs goes unnoticed, uncaught by unitests, overlooked by QA and reach production. Sometimes it's just a minor issue, sometimes it's major bug that costs you lost data and users confidence. Whatever it is, the issue gets fixed. You patch the bug, mend corrupt data and everything get back to it's place.
+Developers often make mistakes. Bugs go unnoticed, uncaught by unit tests, overlooked by QA and reach production. Sometimes it's a just a minor issue, sometimes it's major bug that costs you loose data and users' confidence. Whatever it is, the issue gets fixed. You patch the bug, mend corrupt data and everything gets back to it's place.
 
-Surly there isn't a bug that is unfixable right?
+Surely there isn't a bug that is unfixable right?
 
 Our snippet
 ------------
-As a prologue to this story I'll tell you about our snippet. To use [Jaco](https://getjaco.com), just sign up, copy a small snippet to your site, and your users sessions starts to get recorded. The main goal of the snippet is this:
+As a prologue to this story I'll tell you about our snippet. To use [Jaco] - (https://getjaco.com), just sign up, copy a small snippet to your site, and your users sessions start to be recorded. The main goal of the snippet is this:
```

Comment:
I'm not sure if its "start to get recorded" or "start to be recorded". Both sound weird...